Automattic/wp-calypso
# A4A: Implement Shopping cart in A4A Marketplace.

This pull request adds the functionality to show the shopping cart toggle in the A4A Marketplace. Note that the shopping cart state does not persist and is shared among different pages. This will be addressed on a separate PR.


https://github.com/Automattic/wp-calypso/assets/56598660/ee6bbbfd-c4f3-4bcb-98e7-44efc2045374


Closes https://github.com/Automattic/jetpack-genesis/issues/284

## Proposed Changes

* Implement the ShoppingCart component based on the A4A Marketplace design spec.

## Testing Instructions

* Switch branch: `git checkout add/a4a-mark…hopping-cart`.
* Start the server by running `yarn start-a8c-for-agencies`.
* Go to http://agencies.localhost:3000/marketplace
* Confirm that the `ShoppingCart` component is rendered and behaves correctly.

## Pre-merge Checklist
